tests/unit/dataframer/test_dataframer_smmart.py

```diff
@@ -1,133 +1,257 @@
-        for file in fixture_path.glob('*.ndjson'):
+        
+        assert smmart_fixture_path.exists(), f"Fixture path {smmart_fixture_path.absolute()} does not exist."
+        for file in smmart_fixture_path.glob('*.ndjson'):
             # print(f"Loading {file}")
             for resource in read_ndjson_file(str(file)):
                 db.insert_data_from_dict(resource)
         yield db
 
 
 @pytest.fixture()
-def document_reference_dataframe():
+def smmart_docref_dataframe():
     """Create user-defined dataframe/row with Observations (and user-defined nomenclature) that have focus/association to this DocumentReference instance"""
     return {'identifier': '9ae7e542-767f-4b03-a854-7ceed17152cb',
             'resourceType': 'DocumentReference',
             'id': '9ae7e542-767f-4b03-a854-7ceed17152cb',
-            'status': 'current', 'docStatus': 'final',
+            'status': 'current',
+            'docStatus': 'final',
             'date': '2024-08-21T10:53:00+00:00',
             'md5': '227f0a5379362d42eaa1814cfc0101b8',
-            'title': 'specimen_1234_labA.fq.gz',
             'source_path': 'file:///home/LabA/specimen_1234_labA.fq.gz',
             'contentType': 'text/fastq',
-            'size': 5595609484, 'title': 'specimen_1234_labA.fq.gz',
+            'url': 'file:///home/LabA/specimen_1234_labA.fq.gz',
+            'size': 5595609484,
+            'title': 'specimen_1234_labA.fq.gz',
             'creation': '2024-08-21T10:53:00+00:00',
             'sequencer': 'Illumina Seq 1000',
             'index': '100bp Single index', 'type': 'Exome',
             'project_id': 'labA_projectXYZ',
             'read_length': '100',
             'instrument_run_id': '234_ABC_1_8899',
             'capture_bait_set': 'Human Exom 2X',
-            'end_type': 'Paired-End', 'capture': 'emitter XT',
+            'end_type': 'Paired-End',
+            'capture': 'emitter XT',
             'sequencing_site': 'AdvancedGeneExom',
             'construction': 'library_construction',
-            'Gene': 'TP53', 'Chromosome': 'chr17',
+            'Gene': 'TP53',
+            'Chromosome': 'chr17',
             'result': 'gain of function (GOF)',
             'specimen_collection': 'Breast',
             'specimen_id': '60c67a06-ea2d-4d24-9249-418dc77a16a9',
             'specimen_identifier': 'specimen_1234_labA',
-            'specimen_processing': 'Double-Spun',
-            'url': 'file:///home/LabA/specimen_1234_labA.fq.gz'
+            'specimen_processing': 'Double-Spun'
             }
 
+@pytest.fixture()
+def smmart_observation_dataframes():
+    """Create user-defined dataframe/row with Observations (and user-defined nomenclature) that have focus/association to this DocumentReference instance"""
+    return [{
+        "identifier": "patientX_1234-9ae7e542-767f-4b03-a854-7ceed17152cb-sequencer",
+        "resourceType": "Observation",
```

Comment:
Unclear why we are creating a dataframe on an observation?

Reply:
Agreed, see above. Willing to delete these if we get rid of observation flattening functionality

Reply:
@quinnwai Yes, please delete.

Reply:
deleted, see [recent commit](https://github.com/ACED-IDP/gen3_util/pull/93/commits/1337ac77e8552d06b4c0eba29f4dec9b6c4f1fa4) and also deleted observation flattening in [fhir_import_export.py](https://github.com/ACED-IDP/aced_etl_pod/pull/37/commits/ff83f4d1ba28ed421b7e929fdc7b6f7ec99ef4db) side. Tested data submission locally with new code, works as expected.